include/revng/Model/CallEdge.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Model/AbstractFunctionEdge.h"
+#include "revng/Model/VerifyHelper.h"
+
+/* MODEL-YAML
+name: CallEdge
+doc: A CFG edge to represent function calls (direct, indirect and tail calls)
+type: struct
+inherits: FunctionEdge
+fields:
+  - name: Prototype
+    doc: |
+      Path to the prototype for this call site
+
+      In case of a direct function call, it has to be invalid.
+    # TODO: how can we handle this?
```

Comment:
Handle what?

Reply:
It's a leftover TODO, marking that I needed to figure out how to handle `TupleTreeReference`s